### DISCUSSION TITLE
Month and day are not on one line

### DISCUSSION THREAD

**Discussion by ceromanc** [totaldebug/atomic-calendar-revive#556] [Support]
Hello,

I have an issue that the day and month are not on one line in events mode. It looks, like there was already an issue #281 but it is solved and should work with both US and EU date format, but it doesn't. Thema takes no effect, as well. Does anyone know how to fix this? So each event takes up a lot of space.

Thanks in advance :)

My current code:

`type: custom:atomic-calendar-revive
name: Calendar Card
enableModeChange: true
entities: calendar.hausassistant_gmail_com
maxDaysToShow: 100
maxEventCount: 4
disableEventLink: true
disableLocationLink: true
showRelativeTime: true
showCurrentEventLine: false
showProgressBar: false
showWeekDay: true
showMonth: true
showHiddenText: true
dayWrapperLineColor: transparent
europeanDate: true`

![US format](https://user-images.githubusercontent.com/96552409/151963401-a523b544-21dc-49fe-bfc9-89ca945ed81f.png)
![EU format](https://user-images.githubusercontent.com/96552409/151963405-7048c03c-8528-4c87-88c8-e4a6705de7aa.png)

**Comment by jankete** [comment]
The same here. It would also be great, if we could fix the size (width) of the date column or choose if it should be displayed in 1, 2 or 3 lines. This could save a lot of vertical space in some cases.

![image](https://user-images.githubusercontent.com/22338766/167780703-e12b9ba1-eb8c-49b9-8f40-e5c72a0f1317.png)

**Comment by marksie1988** [comment]
I have now implemented the compact view that will squash everything down much more this may suit your needs more. 

Currently by design the day, date and month are in different div's on new lines due to causing issues with width when all are enabled with other options in the card.
